pkg/logs/message/message.go

```diff
@@ -5,25 +5,46 @@
 
 package message
 
+import "github.com/DataDog/datadog-agent/pkg/logs/config"
+
 // Message represents a log line sent to datadog, with its metadata
 type Message struct {
 	Content    []byte
 	Origin     *Origin
-	status     string
-	Timestamp  string
 	RawDataLen int
+	Timestamp  string
+
+	status string
+}
+
+// NewPartialMessage3 constructs message with content, status and origin.
+func NewPartialMessage3(content []byte, status string, origin *Origin) *Message {
```

Comment:
Why did we add all those constructors ? Plus the naming is not clear, I'd avoid doing that.

Reply:
The reason behind these constructors is to be more clear of what exact fields are passed in. To Avoid taking nil as input. centralize the assignments of all fields.